shared/src/main/java/uk/gov/di/authentication/shared/entity/UpdateClientConfigRequest.java

```diff
@@ -30,6 +30,9 @@
     @JsonProperty("service_type")
     private String serviceType;
 
+    @JsonProperty("request_uris")
+    private List<String> requestUris;
```

Comment:
Would `List<URI>` be more appropriate?

Reply:
There's a good argument for that in both this class and the ClientRegistrationRequest. I kept it consistent with how we have list other attributes that could contain URIs. Maybe there's a refactoring needed here but leaving it out of scope for this change